### ISSUE TITLE
Gradient Accumulation and FP16 for Demucs

### ISSUE THREAD

**Issue by lyndonlauder** [facebookresearch/demucs#389]
Hello,

If possible please can you add an option or advise me for Gradient Accumulation, with my gpu i can only use the batch size 8 and my model loss stops early because of this. I would like to try 64 batch size like your paper. I also request that you please add an option for FP16 training.

Thanks for this great work!
